docs/topics/development/waffle.md

```diff
@@ -0,0 +1,127 @@
+# Waffle
+
+We use [waffle](https://waffle.readthedocs.io/en/stable/) for managing feature access in production.
+
+## Why switches and not flags
+
+We prefer to use [switches](https://waffle.readthedocs.io/en/stable/types/switch.html)
+over flags, as switches are:
+
+- switches are simple
+- switches are easy to reason about
+
```

Comment:
Maybe we could have a sentence about flags there. They can be used in some cases, if you want to do a gradual rollout for instance. We currently have one in production right now (`2fa-enforcement-for-developers-and-special-users`)